politeiawww/cmd/cmswww/updategithub.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+
+	v1 "github.com/decred/politeia/politeiawww/api/cms/v1"
+	"github.com/decred/politeia/politeiawww/cmd/shared"
+)
+
+// UpdateGithubCmd
+type UpdateGithubCmd struct {
```

Comment:
Is there any reason to not accept month/year here and set it to current if missing like you are doing with the `codestats`? If I run `cmswww updategithub --repo=politeiagui decred` I get `GET /v1/admin/updategithub?Month=0&OnlyCodeStats=false&Organization=decred&Repository=politeiagui&Year=0 HTTP/1.1 Internal error 1594670190: handleUpdateGithub: processUpdateGithub: http returned 403`.

Reply:
Month and Year are defaulted to 0

Reply:
mmmm 403 means an auth error or something.  I dont think it's related to the request.  You should be able to not populate month/year there, it's totally optional.

Reply:
hmm, weird, I'm not getting the 403 anymore. It was a brand new access token, maybe there's some kind of delay. Now I'm getting:

```
2020-07-13 22:36:58.281 [INF] GHTR: dcrlnd
2020-07-13 22:36:58.281 [INF] GHTR: Syncing decred/dcrlnd
2020-07-13 22:39:07.708 [ERR] PWWW: 127.0.0.1:53151 GET /v1/admin/updategithub?Month=0&OnlyCodeStats=false&Organization=decred&Repository=&Year=0 HTTP/1.1 Internal error 1594690747: handleUpdateGithub: processUpdateGithub: json: cannot unmarshal object into Go value of type []api.ApiPullRequestCommit
```

I ran the exact command from above `cmswww updategithub --repo=politeiagui decred`. It's weird that I specified the `politeiagui` repo and it continues to update github data from other repos, in this case, `dcrlnd`. Before this error on `dcrlnd` the stats from `politeiagui` repo synced correctly.